sdk/formrecognizer/ai-form-recognizer/MIGRATION-v3_v4.md

```diff
@@ -233,7 +233,7 @@ if (table.boundingBox) {
 }
 ```
 
-Current (4.0.0-beta.1):
+Current (4.0.0-beta.2):
```

Comment:
NIT: does the specific version needs to be mentioned here? Perhaps just v4 can work so you do not have to update these lines every time you make a release.

Reply:
Good idea. Since this is in the migration guide and not embedded into the package, I'll defer this until after the release.